app/helpers/brexit_checker_helper.rb

```diff
@@ -91,4 +91,23 @@ def brexit_results_description(actions, criteria_keys)
       t("brexit_checker.results.description_no_answers").html_safe
     end
   end
+
+  def account_signup_jwt(criteria_keys)
+    account_jwt = BrexitChecker::AccountJwt.new(criteria_keys)
+    account_jwt.encode
+  end
+
+  def path_based_on_account_feature_flag
+    if Rails.configuration.feature_flag_govuk_accounts
+      transition_checker_save_results_path(c: criteria_keys)
+    else
+      transition_checker_email_signup_path(c: criteria_keys)
+    end
+  end
+
+  def redirect_away_from_accounts
+    unless Rails.configuration.feature_flag_govuk_accounts
+      render file: "#{Rails.root}/public/404.html", status: :not_found
```

Comment:
:thinking: If I remove that line and then try to go to
`http://finder-frontend.dev.gov.uk/transition-check/save-your-results?c[]=living-ie` without the feature flag set, instead of getting a 404 i get an error that suggests it has tried to render the page...

Do i need `plain:` or something in there too?
You mind testing this line and seeing if you get the same?

Reply:
Ah, yeah, I forgot that you need to actually return some body content too.  In which case it doesn't matter what we return as nginx will serve a GOV.UK error anyway.  Happy to leave as the 404.html file then.